app/story/story.json

```diff
@@ -130,25 +117,23 @@
             ],
             "paths": {
                 "Oh mein Gott! Es gab Schüsse!": "observe_suspect.suspect_dead"
-            },
-            "tasks": []
+            }
         },
         "observe_suspect.suspect_dead": {
             "description": [
                 "Verdammt! Können Sie mehr sehen?"
             ],
             "paths": {
                 "Ja... {suspect} ist tot.": "end"
-            },
-            "tasks": []
+            }
         },
         "end": {
             "description": [
                 "Hergott! Es sieht so aus als wäre das nur ein kleiner Fisch in einem viel gößeren Fall gewesen...",
                 "Kommissar {user_name}, es sieht so aus als haben wir es hier mit der Mafia zu tun - ich kann ihnen nur Glück wünschen..."
             ],
             "paths": {},
-            "tasks": []
+            "actions": ["telegram_highjack"]
```

Comment:
Is this supposed not to go off during the mock playthough?
Cant test it properly...

Reply:
the telegram highjack of course only does work with the app (we could also mock it, but that would make it very unimpressive, because you would manually have to send the code)
The reason it doesn't go off is probably because there is no phonenumber set. If you do that manually, there should be a login attempt.